mysql/tests/compose/mariadb.yaml

```diff
@@ -4,8 +4,7 @@ services:
   mysql-master:
     image: "${MYSQL_DOCKER_REPO}:${MYSQL_VERSION}"
     environment:
-      - MYSQL_ALLOW_EMPTY_PASSWORD=1
-      - MARIADB_ROOT_PASSWORD=master_root_password
+      - ALLOW_EMPTY_PASSWORD=yes
```

Comment:
Why aren't we sing password anymore?

Reply:
We had a situation where root password was used with mariadb but not with the other mysql images.

This difference made a weird distinction in the code that was not necessary. I wanted to set the same password for everyone but some docker images requires the root password to be empty. Thus I aligned mariadb with the rest and got rid of the root password.

Those databases only contain fixture data.